File: Coding/03_Beginner/Full_search/C++/13.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string S, c;
    cin >> S >> c;
    bool found = false;

    for (char ch : S){
        if (ch == c[0]){
            found = true;
            break;
        }
    }

    cout << (found ? "Yes" : "No") << endl;
    return 0;
}

File: Coding/03_Beginner/Full_search/JavaScript/13.js
const fs = require("fs");
const [S, c] = fs.readFileSync(0, "utf8").trim().split("\n");

let found = false;
for (let i = 0; i < S.length; i++){
    if (S[i] == c){
        found = true;
        break;
    }
}

console.log(found ? "Yes" : "No");